src/lib/get-topper-settings.js

```diff
@@ -98,12 +98,21 @@ const usePackageTopper = (content) => {
 	};
 };
 
-const useEditoriallySelectedTopper = (content) => {
+const useEditoriallySelectedTopper = (content, flags) => {
 	let backgroundColour;
+	let myFtButton;
 
 	// Convert old palette colours to new palette colours from Methode
 	if (content.topper.layout === 'full-bleed-offset') {
 		backgroundColour = 'wheat';
+	} else if (content.topper.layout === 'deep-landscape') {
+		if (['paper', 'wheat'].includes(content.topper.backgroundColour)) {
+			backgroundColour = 'white'
+		}
+		myFtButton = {
+			variant: 'opinion',
+			followPlusDigestEmail: followPlusDigestEmail(flags)
+		}
```

Comment:
Suggestion: it worth try also to see if the logic about the variant could be moved inside the specific method likely designated for that [myFtButtonVariant](https://github.com/Financial-Times/n-map-content-to-topper/blob/4efb8cab4e473377d7946347ff53bdb503b738b5/src/map-content-to-topper.js#L7)